js/main.js

```diff
@@ -26,8 +26,10 @@ function submitForm(event) {
       notes: $notes.value,
       entryId: data.editing
     });
-
-    entryList.prepend(newEntryCreation(data.editing));
+    updateData(data.editing);
+    var li = entryList.querySelectorAll('li');
+    li[data.editing] = newEntryCreation(data.editing);
```

Comment:
Does this work? `NodeList`s are read-only, so you shouldn't be able to update the page by changing values inside of it. What you really need to do is look through the li elements in the page, find the `li` for this entry being edited, and `replace` that DOM element entirely with the newly generated DOM tree